### PR TITLE
Resolve build issues related to remix and stateful components

### PR DESCRIPTION
**Pull request by schpet**
**don't merge: not in a working state**

- import react!
- Switchup rollup config

### CODE REVIEW

**Comment by schpet** [comment]
thought this was working but unfortunately not the case

File: libs/design-system/src/lib/Components/InternalCodeblock/index.tsx [hunk 1]
```diff
@@ -104,29 +103,6 @@ export function InternalCodeblock({
           )}
         />
       </div>
-      <Dialog
```
Comment (10thfloor) [inline libs/design-system/src/lib/Components/InternalCodeblock/index.tsx:107]: Was this removed intentionally?

**Review by 10thfloor — APPROVED** [review]

File: libs/design-system/src/lib/Components/InternalCodeblock/index.tsx (hunk 1 above)
Comment (schpet) [reply libs/design-system/src/lib/Components/InternalCodeblock/index.tsx:107]: yeah consider this whole thing an experiment that didn't work out. removing this changed the way it failed slightly: without dialog the page will load and errors are logged in the browser console, with the dialog the page doesn't load at all.

**Comment by 10thfloor** [comment]
Closing this. Upcoming work to remove NX and consolidate design-system into Remix app will fix this.

**Pull request closed without merging**